### ISSUE TITLE
typespec.io failing WCAG 4.1.2 (aria-hidden elements are not focusable)

### ISSUE THREAD

**Issue by kyle-patterson** [Azure/typespec-azure#453]
When running the Accessibility Insights for Web tool, the `aria-hidden-focus` rule will fail on typespec.io.

#### Issue
`Ensures aria-hidden elements are not focusable nor contain focusable elements` ([aria-hidden-focus](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus))
#### Target application
[TypeSpec](https://typespec.io/)
#### Element path
i:nth-child(1)
#### Snippet
`<i tabindex="0" role="none" data-tabster-dummy="" aria-hidden="true" style="position: fixed; height: 1px; width: 1px; opacity: 0.001; z-index: -1; content-visibility: hidden; top: 0px; left: 0px;"></i>`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Microsoft Edge version 122.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by kyle-patterson** [comment]
This appears to actually be a bug in the scanner - the "issue" itself is by design. Marked the internal utility as such, closing this out.
